projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -930,12 +930,10 @@ public function get_attachment_comments() {
 		 */
 		do_action( 'jp_carousel_check_blog_user_privileges' );
 
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended -- we do not need to verify the nonce for this public request for publicly accessible data (as checked below).
```

Comment:
Great detail. This will save someone in the future.

Reply:
Absolutely. Thanks @jeherve for adding that in the review suggestion!